packages/stylelint-plugin-ckeditor5-rules/lib/index.js

```diff
@@ -0,0 +1,13 @@
+/**
```

Comment:
It seems that `stylelint` does not understand the concept of the file. I'd remove it and move all `rules/*` to the `lib/` directory without the `index.js` file.